pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

```diff
@@ -76,10 +56,5 @@ kubectl delete -n kubeaddons service gatekeeper-webhook-service
 
 ```
 
-<p class="message--note"><strong>WARNING: </strong>
-As long as the secret exists, <code>helm list -n kubeaddons</code>
-will wrongly report existence of a Helm release called "gatekeeper-kubeaddons".
-If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper
-Platform Application in Kommander, potentially making Kubernetes on the cluster
-inoperable.
-</p>
+<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code>
+will wrongly report existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```

Comment:
```suggestion
wrongly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```